controllers/kubevirtmachine_controller_test.go

```diff
@@ -140,6 +156,8 @@ var _ = Describe("reconcile a kubevirt machine", func() {
 	mockCtrl = gomock.NewController(GinkgoT())
 	workloadClusterMock := workloadclustermock.NewMockWorkloadCluster(mockCtrl)
 	infraClusterMock := infraclustermock.NewMockInfraCluster(mockCtrl)
+	machineFactoryMock := machinemocks.NewMockMachineFactory(mockCtrl)
+	machineMock := machinemocks.NewMockMachineInterface(mockCtrl)
```

Comment:
these two variables are getting deleted in BeforeEach, so you can just define them with `var` here, without initiate them.

```suggestion
var machineFactoryMock *MockMachineFactory
var machineMock *MockMachineInterface
```